addon/components/bs-accordion/item/title.hbs

```diff
@@ -1,23 +1,40 @@
 {{!-- template-lint-disable no-nested-interactive --}}
 {{!-- @todo fix this, see https://github.com/kaliber5/ember-bootstrap/issues/999 --}}
-<div
-  class="{{if @collapsed "collapsed" "expanded"}} {{if (macroCondition (macroGetOwnConfig "isNotBS3")) "card-header"}} {{if (macroCondition (macroGetOwnConfig "isBS3")) "panel-heading"}}"
-  role="tab"
-  ...attributes
-  {{on "click" this.handleClick}}
->
-  {{#if (macroCondition (macroGetOwnConfig "isNotBS3"))}}
-    <h5 class="mb-0">
-      <button class="btn btn-link {{if @disabled "disabled"}}" type="button" disabled={{@disabled}}>
-        {{yield}}
-      </button>
-    </h5>
-  {{/if}}
-  {{#if (macroCondition (macroGetOwnConfig "isBS3"))}}
-    <h4 class="panel-title">
-      <a href="#" class={{if @disabled "disabled"}}>
-        {{yield}}
-      </a>
-    </h4>
-  {{/if}}
-</div>
\ No newline at end of file
+{{#if (macroCondition (macroGetOwnConfig "isBS5"))}}
+  <h2
+    class="accordion-header"
+    role="tab"
+    ...attributes
+  >
+    <button
+      class="accordion-button {{if @disabled "disabled"}} {{if @collapsed "collapsed"}}"
+      type="button"
+      disabled={{@disabled}}
+      {{on "click" this.handleClick}}
+    >
+      {{yield}}
+    </button>
+  </h2>
+{{else}}
+  <div
+    class="{{if (macroCondition (macroGetOwnConfig "isBS3")) "panel-heading"}} {{if (macroCondition (macroGetOwnConfig "isBS4")) "card-header"}}"
+    role="tab"
+    ...attributes
+    {{on "click" this.handleClick}}
+  >
+    {{#if (macroCondition (macroGetOwnConfig "isBS4"))}}
+      <h5 class="mb-0">
+        <button class="btn btn-link {{if @disabled "disabled"}} {{if @collapsed "collapsed"}}" type="button" disabled={{@disabled}}>
```

Comment:
Had to change BS3/4 markup a little, putting `.collapsed` on the button/a instead of the parent div. This is actually expected (see e.g. https://getbootstrap.com/docs/4.6/components/collapse/#accordion-example), so could be seen here as a bug fix. Though it might theoretically cause some regressions for users... 🤔

It does not affect the CSS styling in any way for BS3/4 AFAICT, but it does for BS5, so had to be changed.

Reply:
> So I guess we don't need to address that bug here.

You are right. Extracted the changes to #1590.

Will rebase this PR once the fix has landed.

Reply:
@jelhan done now! The diff is now much smaller, and confined to only BS5 related changes!